renovate.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
+    ":automergeRequireAllStatusChecks",
+    ":automergeStableNonMajor",
+    ":automergeLinters",
+    ":automergeTesters",
+    ":automergeTypes",
+    ":enableVulnerabilityAlerts",
+    ":combinePatchMinorReleases",
+    ":ignoreUnstable",
+    ":maintainLockFilesWeekly",
+    ":pinAllExceptPeerDependencies",
+    ":rebaseStalePrs"
+  ],
+  "labels": ["dependencies"],
+  "branchPrefix": "renovate-action/",
+  "prConcurrentLimit": 2,
+  "packageRules": []
+}
```

Comment:
No need for `schedule:` config here, we already have `cron` set up in the GH action YML file and that should be sufficient to run Renovate (https://docs.renovatebot.com/key-concepts/scheduling/#global-schedule-vs-specific-schedule)

Reply:
makes sense 👍